config/locales/diaspora/en.yml

```diff
@@ -851,12 +851,15 @@ en:
       message: |-
         Hello!
 
-        You have been invited to join diaspora*!
+        You have been invited to join diaspora* by %{diaspora_id}!
 
         Click this link to get started
 
         [%{invite_url}][1]
 
+        You can add %{diaspora_id} to the pod you might be currently using.
```

Comment:
Please let me know if this is a correct description. Also, I can't understand how to handle the case where `diaspora_id` is `nil`.